.github/codeql-config.yml

```diff
@@ -0,0 +1,11 @@
+query-filters:
+  - exclude:
+      id: go/incorrect-integer-conversion
+  - exclude:
+      id: go/reflected-xss
+  - exclude:
+      id: go/allocation-size-overflow
+  # - exclude:
```

Comment:
@jianjuns @tnqn @antoninbas could you take a look at this PR? The purpose is to enable code scanning for Antrea since LGTM.com is sunset. I noticed there was some comments like ` lgtm[go/reflected-xss]` to disable some checks, so I disabled a few as well.  But there are also a few new alerts by code scanning. Could you check if we can disable them as well or we need to fix it? thanks.

Reply:
I am fine if you want to exclude log injection.

Since unsafe quoting is reported as _critical_, maybe you could check if there is a way to address it

Reply:
@antoninbas after check with @wenyingd , seems there is no easy way to address `unsafe quoting` warning. In Windows, when antrea create a vNIC named `pod1-abcdef`, the corresponding NetAdapter's name will be like `vEthernet (pod1-abcdef)` which has a space, so we have to use double quotes to run command correctly. The two warnings are all from Windows commands, I feel it's OK to exclude `unsafe-quoting` check. I will update the codeql-config.yml, let me know if you have any concern.

Reply:
why exclude it globally instead of dismissing these specific alerts manually?

Reply:
Oh, yes, you are right, let me update the config file and dismiss one by one. thanks.

Reply:
Done